### PR TITLE
Styles

### PR DESCRIPTION
**Pull request by azevedo-252**
Using *.scss

Added:
* autoprefixer
* normalize.css
* gulp-import-css (for being able to import *.css files)

Created some CSS components:
* Site layout
* Grid

### CODE REVIEW

File: app/index.html [hunk 1]
```diff
@@ -10,9 +10,9 @@
 
     <link rel="stylesheet" href="css/main.css">
   </head>
-  <body ng-app='theshelf'>
-    <side-content></side-content>
-    <div ui-view></div>
+  <body class='Site' ng-app='theshelf'>
```
Comment (gabrielpoca) [inline app/index.html]: Wasn't your intention to use the old css?

Comment (azevedo-252) [inline app/index.html]: Maybe at some point. But I changed my mind :)

Comment (gabrielpoca) [inline app/index.html]: And why call it `Site`? What happens if you have another one of these? What will you call it?

Comment (azevedo-252) [inline app/index.html]: Well there isn't any indication that there will be any.
I felt like simplifying.
What would you call it anyway? It's the site's layout.
And besides, it's used in one place alone (here), if the need comes to change it, then it couldn't be easier.

Comment (gabrielpoca) [inline app/index.html]: It's the site our the site layout?

Comment (azevedo-252) [inline app/index.html]: It's supposed to be the same.
I don't believe calling *Site* or *Layout* makes much difference now.
I'm just experimenting, so unless the name causes confusion, I'd say we could go with this.

**Comment by gabrielpoca** [comment]
:+1:

**Pull request merged**